tmca-tester/tmca-tester.js

```diff
@@ -17,7 +17,7 @@
 
 const fs    = require('fs');
 const https  = require('https');
-var menu  = require('node-menu');
+const readline = require('readline');
```

Comment:
Issue (blocking): If node-menu is no longer required and has been replaced with readline, the README should be updated accordingly: https://github.com/CDOT-CV/jpo-security-svcs/blob/Update/dependencies/tmca-tester/README.md

Reply:
The README has been updated to reflect these changes